services/QuillCMS/spec/models/graded_response_spec.rb

```diff
@@ -2,9 +2,9 @@
 
 RSpec.describe GradedResponse, type: :model do
   context 'basic queries' do
-    let!(:response_optimal) {create(:response, question_uid: '123', optimal: true)}
-    let!(:response_nonoptimal) {create(:response, question_uid: '123', optimal: false)}
-    let!(:response_ungraded) {create(:response, question_uid: '123', optimal: nil)}
+    let!(:optimal) {create(:response, question_uid: '123', optimal: true)}
```

Comment:
Maybe time for factory aliases for optimal_response, non_optimal_response, etc.

Reply:
Hmm, maybe. I'm a bit on the fence here, since being explicit inline is about the same length as the abstraction (and some of the names can be confusing, i.e. distinguishing between `optimal` `false` and `nil`, which many times are treated the same). Let me think about this.

```ruby
create(:optimal_response)
create(:response, optimal: true)

Reply:
While length is about the same,  `create(:optimal_response)` prevents the accidentally specifying wrong value for `optimal`.  It also makes the `let(:optimal) { create(:optimal_response) }` easier to read.

Reply:
One other point is that it provides and opportunity to standardize language around what's being created.

Reply:
Ok, you've made several good points, I'll add the aliases. Thanks for the feedback on this.